build/pkgs/tachyon/patches/Make-arch.patch

```diff
@@ -1,844 +0,0 @@
---- a/unix/Make-arch
```

Comment:
```suggestion
diff --git a/unix/Make-arch b/unix/Make-arch
index 08afb85..dbeb691 100644
--- a/unix/Make-arch
+++ b/unix/Make-arch
@@ -1471,6 +1471,25 @@
 	"RANLIB = ranlib" \
 	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread"
 
+# Linux Arm using gcc
+linux-arm:
+	$(MAKE) all \
+	"ARCH = linux-arm" \
+	"CFLAGS = -Wall -O3 -fomit-frame-pointer -ffast-math -DLinux $(MISCFLAGS)" \
+	"ARFLAGS = r" \
+	"STRIP = strip" \
+	"RANLIB = ranlib" \
+	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread"
+
+# Linux Arm using gcc, with threads
+linux-arm-thr:
+	$(MAKE) all \
+	"ARCH = linux-arm-thr" \
+	"CFLAGS = -Wall -O3 -fomit-frame-pointer -ffast-math -DLinux -DTHR -D_REENTRANT $(MISCFLAGS)" \
+	"ARFLAGS = r" \
+	"STRIP = strip" \
+	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread"
+
 # Linux x86 using gcc, threads, and OpenGL
 linux-thr-ogl:
 	$(MAKE) all \
```

I needed this to successfully build this PR on a Macbook Air M2 running GNU/Linux. Not sure if `linux-arm` part is needed since for me the determined `ARCH` seems to be `linux-arm-thr` but I haven't tried without it.

Reply:
could you send this as a PR against my branch please